content/riak/kv/3.2.5/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Riak KV 3.2.5"
+description: ""
+project: "riak_kv"
+project_version: "3.2.5"
+lastmod: 2022-12-30T00:00:00-00:00
+sitemap:
+  priority: 0.9
+menu:
+  riak_kv-3.2.5:
+    name: "Riak KV"
+    identifier: "index"
+    weight: 100
+    pre: riak
+toc: false
+aliases:
+  - /riak/3.2.5/
+---
+
+[aboutenterprise]: https://www.tiot.jp/en/about-us/contact-us/
+[config index]: {{<baseurl>}}riak/kv/3.2.5/configuring
+[downloads]: {{<baseurl>}}riak/kv/3.2.5/downloads/
+[install index]: {{<baseurl>}}riak/kv/3.2.5/setup/installing/
+[plan index]: {{<baseurl>}}riak/kv/3.2.5/setup/planning
+[perf open files]: {{<baseurl>}}riak/kv/3.2.5/using/performance/open-files-limit
+[install debian & ubuntu]: {{<baseurl>}}riak/kv/3.2.5/setup/installing/debian-ubuntu
+[getting started]: {{<baseurl>}}riak/kv/3.2.5/developing/getting-started
+[dev client libraries]: {{<baseurl>}}riak/kv/3.2.5/developing/client-libraries
+
+Riak KV is a distributed NoSQL database designed to deliver maximum data availability by distributing data across multiple servers. As long as your Riak KV client can reach one Riak server, it should be able to write data.
+
+This release is tested with OTP 20, OTP 21 and OTP 22; but optimal performance is likely to be achieved when using OTP 22.
+
+## Supported Operating Systems
+
+- Alpine Linux 3.21
+- Amazon Linux 2023
+- CentOS 8
+- CentOS 9
+- Debian 10.0 ("Buster")
+- Debian 11.0 ("Bullseye")
+- Debian 12.0 ("Bookworm")
+- Oracle Linux 9
+- Red Hat Enterprise Linux 8
+- Red Hat Enterprise Linux 9
+- Raspbian Bullseye
```

Comment:
There's no Raspsian release on the file store. Should there be, and this is right? Or should there not be and this is wrong?

Reply:
Also one I was going to ask Adriy about